tests/product/test_control.py

```diff
@@ -65,37 +65,10 @@ def test_server_start_one_host_started(self):
         self.assert_start_with_one_host_started(
             self.cluster.internal_slaves[0])
 
-    def test_server_start_all_started(self):
-        self.setup_cluster(NoHadoopBareImageProvider(), STANDALONE_PRESTO_CLUSTER)
-        start_output = self.run_prestoadmin('server start').splitlines()
-        process_per_host = self.get_process_per_host(start_output)
-        self.assert_started(process_per_host)
-
-        start_output = self.run_prestoadmin(
-            'server start', raise_error=False).splitlines()
-        self.assertRegexpMatchesLineByLine(
-            start_output,
-            self.expected_port_error(self.cluster.all_internal_hosts())
```

Comment:
We now have no tests that verify that you can't run start twice (e.g. that you get the "port in use" error)